src/vm/eventpipe.cpp

```diff
@@ -219,10 +219,13 @@ void EventPipe::EnableOnStartup()
         SString outputPath;
         outputPath.Printf("Process-%d.netperf", GetCurrentProcessId());
 
+        // Get the circular buffer size in megabytes.
+        unsigned int circularMB = CLRConfig::GetConfigValue(CLRConfig::INTERNAL_EventPipeCircularMB);
```

Comment:
Ideally all of this just gets pushed to managed code and it is there where config values are fetched and set as needed.   Native code should not know about config at all.

Reply:
Yup, done.